### ISSUE TITLE
french manual

### ISSUE THREAD

**Issue by llegoff** [alexrj/Slic3r-Manual#18]
Hello, I translate into French the slic3r manual
the source code and pdf are avaialble on github.com/llegoff/Slic3r-Manual-FR
it is interesting that this version to be present in manual.slic3r.org

**Comment by lordofhyphens** [comment]
@llegoff are you interested in doing an update to your translation (the Manual has been mostly turned into Markdown now) and keeping it updated?

**Comment by vivi3n** [comment]
Hey ! I'm little bit noob with github and markdown but I know well 3d print' things. I can translate the manual if you are yet interested. But I would like create my own G-code' interpretor for 3d printers, so can you help me for that? I know G-code but I don't know how build my own g-code' translator for 3d print some "different" design... I like wireprinting method (that I have documented recently) but I would like create more crazy things.. At soon

**Comment by lordofhyphens** [comment]
@vivi3n talking about code should go over to slic3r/Slic3r

We have an issue for wireframe printing, do a search over there and jump I

**Comment by supermerill** [comment]
I can also write french. What's needed to do? Do we create an en and fr folder?

**Comment by llegoff** [comment]
Bonjour, j'ai traduis le manuel, il y a quelques années, les sources sont disponibles sur mon github (https://github.com/llegoff/Slic3r-Manual-FR), il reste à les convertir de Latex vers markdown

**Comment by vivi3n** [comment]
@lordofhyphens yep I know the issue on wireframe, but for moment the guy who have start this, is disappear... And it's funny, nobody answer during more 4 years for translate and suddenly people are interested for do it... ahh guys...

**Comment by vivi3n** [comment]
> @vivi3n talking about code should go over to slic3r/Slic3r
> 
> We have an issue for wireframe printing, do a search over there and jump I

Well, I have reading again the exange about wireframe class (https://github.com/slic3r/Slic3r/issues/3933), but I'm not sure to understand how use the adding in the class. It's a C++ software built out of slic3r no? For the method I can explain all parameter used by Cura with the experimental mode and how it's write in G-code, but I don't know coding like a programmer ( I'm DIY programmer and not good - cause I don't programming all time and just arduino things).

**Comment by vivi3n** [comment]
just for information, I have do documentation on wireprint (by Cura sorry) here : https://www.thingiverse.com/thing:3396739
